.github/workflows/ci.yaml

```diff
@@ -34,6 +34,10 @@ jobs:
       - name: Build
         run: go build -race  ./...
       - name: Test
+        if: matrix.os != 'windows-latest'
+        run: sudo -E PATH="$PATH" bash -c "go test -race ./..."
+      - name: Test
+        if: matrix.os == 'windows-latest'
```

Comment:
Could you explain why this change?

Reply:
this test https://github.com/Mixaster995/sdk-vpp/blob/abstract-sockets/pkg/tools/proxy/proxy_test.go#L44
failes without these lines

Reply:
Got it :)